<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vessel Marker v1.01</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="PenWidthSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>11</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>笔刷大小:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mask透明度:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>15</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>视角跟随</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="MaskComboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>0%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100%</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>15</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>拖拽模式</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="AnnotationView">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="PaintView" name="ImageView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>15</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tips:  ctrl+左键 画图</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_image"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_image">
   <property name="text">
    <string>打开image和mask</string>
   </property>
  </action>
  <action name="action_mask">
   <property name="text">
    <string>打开mask</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>存储 标记图片</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>存储 透明png</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintView</class>
   <extends>QGraphicsView</extends>
   <header>PaintView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
